src/images/local_images.py

```diff
@@ -0,0 +1,110 @@
+from pathlib import Path
+
+from geopy import Point
+from geopy.distance import ELLIPSOIDS, distance
+from loguru import logger as log
+import numpy as np
+from PIL.ExifTags import GPS
+from PIL.Image import open
```

Comment:
It's better to do 

```python
from PIL import Image

Image.open(...)
```

`open` is a built-in function for opening files and shadowing it with another implementation with the same name is really confusing.

Reply:
I renamed `open` to `open_image`, I prefer to only import the specific modules that are being used

Reply:
That's fine. The main thing I think is important is that we don't shadow builtin `open`.

For whatever reason, it's conventional when using Pillow to do `Image.open`: https://pillow.readthedocs.io/en/stable/reference/Image.html#examples

But Pillow has some nonpythonic conventions anyways like naming the `Image` module with a capital I.